cryptoki/src/mechanism/mechanism_info.rs

```diff
@@ -219,6 +218,26 @@ impl MechanismInfo {
     }
 }
 
+impl std::fmt::Display for MechanismInfo {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        let flags = format!("{0:#?}", self.flags);
+        let key_size_info = match self.min_key_size == 0 && self.max_key_size == 0 {
```

Comment:
I think this could be shortened to just `if` instead of `match` with `true`/`false` arms.